common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4053,4 +4058,12 @@ object CelebornConf extends Logging {
       .doc("Kerberos keytab file path for HDFS storage connection.")
       .stringConf
       .createOptional
+
```

Comment:
Spark use configs `NETWORK_CRYPTO_ENABLED` and `SASL_ENCRYPTION_ENABLED` to determine the encryption. Are we going to introduce similar config(s) for Celeborn?

Reply:
We are going to use TLS for encrypting network traffic as proposed in https://docs.google.com/document/d/1D1U2COYhS3ob7l0t2WghRhBk_Fci9RGx-2FBXA3nvXk/edit#heading=h.po9dc3r1kb3k. 
Spark has also added TLS for RPC and shuffle communication recently. Will add TLS support to Celeborn in a future PR.